www/js/splash/updatecheck.js

```diff
@@ -45,6 +45,10 @@ angular.module('emission.splash.updatecheck', ['angularLocalStorage'])
          'Ionic Deploy: Update error! '+ res);
     }, function(prog) {
       console.log('Ionic Deploy: Progress... ', prog);
+      $rootScope.progress = prog;
+      $rootScope.isDownloading = true;
+    }).then(function(){
+      $rootScope.isDownloading = false;
```

Comment:
update does return a promise, but the `then` of that promise doesn't return a promise. In fact, it doesn't return anything (no return statement). Given that, I don't think that the return value of update().then() is itself `thenable`